src/vehicle/models/pod_data.go

```diff
@@ -12,38 +12,40 @@ type PodData struct {
 	Boards map[string]Board `json:"boards"`
 }
 
-func NewPodData() PodData {
-	return PodData{
+func NewPodData() *PodData {
```

Comment:
Why pointer?

Reply:
because AddGlobal and AddPacket work on a pointer to the type